vmm/src/lib.rs

```diff
@@ -2144,8 +2147,8 @@ const DEVICE_MANAGER_SNAPSHOT_ID: &str = "device-manager";
 mod unit_tests {
     use super::*;
     #[cfg(target_arch = "x86_64")]
-    use crate::config::DebugConsoleConfig;
-    use config::{
+    use crate::vm_config::DebugConsoleConfig;
+    use crate::vm_config::{
         ConsoleConfig, ConsoleOutputMode, CpusConfig, HotplugMethod, MemoryConfig, PayloadConfig,
         RngConfig,
     };
```

Comment:
```suggestion
    use crate::vm_config::{
        ConsoleConfig, ConsoleOutputMode, CpusConfig, CpuFeatures, DebugConsoleConfig,
        HotplugMethod, MemoryConfig, PayloadConfig, RngConfig,
    };
```

Reply:
`DebugConsoleConfig`  is only used for `x86_64`.